scripts/generate-acknowledgements.sh

```diff
@@ -21,12 +21,22 @@ fi
 
 # Save licenses
 export TEMPDIR="$(mktemp -d /tmp/generate-acknowledgements.XXXXXX)" || exit 1
-rmdir "${TEMPDIR}"
-"${GO_LICENSES}" save . --save_path="${TEMPDIR}"
+export COMBINEDIR="${TEMPDIR}/combined"
+mkdir -p "${COMBINEDIR}"
+
+# go-licenses output can vary by GOOS.
+# https://github.com/google/go-licenses/issues/187
+# Run it three times for the main OSes, and combine the results.
+for goos in darwin linux windows ; do
```

Comment:
I think we have to put all the GOOSes we publish binaries for in the [release](https://github.com/buildkite/agent/releases) (at least).

By my estimation, that's:
```suggestion
for goos in darwin linux windows freebsd dragonfly netbsd openbsd ; do
```

Keeping this in sync with the buildkite yaml for the release pipeline is also possible with some `yq` like tool.

Reply:
Done (though without any automated syncing).